torch/distributed/rpc/api.py

```diff
@@ -223,9 +223,10 @@ def rpc_sync(to, func, args=None, kwargs=None):
                        invocation.
 
     Returns:
-        Returns the result of running ``func``on ``args`` and ``kwargs``.
+        Returns the result of running ``func`` on ``args`` and ``kwargs``.
 
     Example::
+
```

Comment:
Curious: why do we need the new line here?

Reply:
Adding this makes the code appear nicely and in python format on the documentation page